tests/unittests/unittest.cpp

```diff
@@ -26,6 +29,13 @@
   #include "podio/SIOReader.h"
 #endif
 
+#ifndef PODIO_ENABLE_RNTUPLE
+  #define PODIO_ENABLE_RNTUPLE 0
+#endif
+#if PODIO_ENABLE_RNTUPLE
```

Comment:
Why not just `#ifdef PODIO_ENABLE_RNTUPLE` ? usually the pre-compiler flags are either set or not?

Reply:
Simply copied from the SIO approach, where the library with the definition might not be present. However, the root library is always here, so I will make this always defined to the approriate value.

Reply:
Done.